exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+  "exercise": "armstrong-numbers",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "0 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 0
+      },
+      "expected": true
+    },
+    {
+      "description": "1 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 1
+      },
+      "expected": true
+    },
+    {
+      "description": "9 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 9
+      },
+      "expected": true
+    },
+    {
+      "description": "10 is not an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 10
+      },
+      "expected": false
+    },
+    {
+      "description": "100 is not an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 100
+      },
+      "expected": false
+    },
+    {
+      "description": "153 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 153
+      },
+      "expected": true
+    },
+    {
+      "description": "370 is an Armstrong number",
+      "property": "isArmstrongNumber",
+      "input": {
+        "number": 370
+      },
+      "expected": true
+    },
+    {
+      "description": "9474 is an Armstrong number",
```

Comment:
```text
"description": "Four digit Armstrong number",
```

Reply:
Perhaps a test of a non-Armstrong four digit number would be prudent here as well.